src/style.css

```
@@ -0,0 +1,68 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.form {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 400;
```

```suggestion
  font-weight: 300;
```

remove `font-style: normal;` because it is default value,

also, you don't have  font for weight 400, as you defined only 300 and 800